sklearn/utils/testing.py

```diff
@@ -688,6 +692,29 @@ def func(*args, **kwargs):
     return decorator
 
 
+def if_safe_multiprocessing_with_blas(func):
```

Comment:
Sorry for slow replies.
Why are you not fine with the tests crashing when people used OpenMP?
The tests test if the setup is working correctly, right? If their production system will crash, the tests should crash.